src/plugin/Manager.h

```diff
@@ -311,6 +311,18 @@ class Manager
 	 */
 	bool HookQueueEvent(Event* event) const;
 
+	/**
+	 * Hook that filters the dispatching of an event.
+	 *
+	 * @param event The event to be executed; it may be modified.
```

Comment:
As I read this I wondered if that itself is a use-case — modifying the event at dispatch time vs enqueue time, say for time-tracking or other internal diagnostics.